fineract-provider/build.gradle

```diff
@@ -20,10 +20,17 @@ description = '''\
 Run as:
 gradle clean bootRun'''
 
+group = 'org.apache.fineract'
+buildDir = new File(rootProject.projectDir, "../build")
+
+repositories {
+    jcenter()
+    mavenCentral()
```

Comment:
This shouldn't be required, no? We didn't have it before either

Reply:
Hi @vorburger  - welcome back!

The same lines were there, but just hidden half way down the build.gradle file (lines 164-170 in the previous version). All I did was to reorganise the existing content - I was not trying to make any functional changes at this point, so did not remove any of the settings.

I have another PR ready to further reorganise the content by e.g. grouping all the plugin configurations together. But once we have the contents structured in a more sensible way, then it should be easy to see whether all of the settings are actually needed and remove the ones that are not.

Reply:
SGTM. FYI what I meant was specifically the mavenCentral() line - AFAIK jCenter() "proxies" (mirrors) Maven Central already anyway, so this is unnecessary and probably just shows down (failed) look ups.